doc/rst/source/grdmix.rst

```diff
@@ -0,0 +1,147 @@
+.. index:: ! grdmix
+.. include:: module_core_purpose.rst_
+
+******
+grdmix
+******
+
+|grdmix_purpose|
+
+Synopsis
+--------
+
+.. include:: common_SYN_OPTs.rst_
+
+**gmt grdmix**
+*raster1* [ *raster2* [ *raster3*]]
+|-G|\ *outfile*
+[ |-A|\ *weights* ]
+[ |-C| ]
+[ |-D| ]
+[ |-I|\ *intens* ]
+[ |-M| ]
+[ |-N| ]
+[ |SYN_OPT-V| ]
+[ |SYN_OPT-f| ]
+[ |SYN_OPT--| ]
+
+|No-spaces|
+
+Description
+-----------
+
+**grdmix** will perform various operations involving images and grids.
+We either use a *weights* grid, image, or constant to add a new
+transparency layer to the image given as *raster1*, or we will blend
+the two *raster1* and *raster2* (grids or images) using the *weights* for
+*raster1* and the complementary *1 - weights* for *raster2* and save to
+*outfile*. Alternatively, we will deconstruct an image into its component
+grids or construct an image from its normalized component grids.
+All *raster?*, *weights* and *intens* files must have the same 
+dimensions. The optional *weights* and *intens* files may be replaced by
+constant values instead.
+
+Required Arguments
+------------------
+
+*raster?*
+    If only one is given and **-C** is not set then *raster1* must be an image.
+    If two are given then *raster1* and *raster2* must both be either
+    images or grids.  If three are given then they must all be grids and
+    **-C** must be set.
+
+.. _-G:
+
+**-G**\ *outfile*
+    The name for the output raster.
```

Comment:
Document the supported image formats.

Reply:
Thanks, I guess use the same stuff as for grdimage -A, @joa-quim ? Or for this (we are not writing grids) maybe just list the standard image formats geotiff, png, jpg, ???

Reply:
It should be able to read all GDAL formats. But for the *standard* we do some guessings in grdimage to avoid having to state de driver name, I think.